command/ca/acme/acme.go

```diff
@@ -0,0 +1,19 @@
+package acme
+
+import (
+	"github.com/smallstep/cli/command/ca/acme/eab"
+	"github.com/urfave/cli"
+)
+
+// Command returns the acme subcommand.
+func Command() cli.Command {
+	return cli.Command{
+		Name:        "acme",
+		Usage:       "manage ACME EAB",
+		UsageText:   "**step beta ca acme** <subcommand> [arguments] [global-flags] [subcommand-flags]",
```

Comment:
These aren't under `beta` are they?

Reply:
oh, nvm. They are under `beta`. Is that a good idea? Why `beta` as opposed to normal? Are we still figuring out the API?

Reply:
I just followed your example ;-)

More seriously: my thinking back then was that new commands would first be `beta` and somewhat later `beta` would be removed. Another thing that was on my mind back then was the fact that the DB backed provisioners were something new and this seemed to also (always) needed to be DB backed. 

I'd say the API, commands, etc. are most probably done, so in that sense they're not really `beta`.

Reply:
We can leave it in beta for now if you like. Should be easy to migrate later since there's no conflic.

Reply:
Yes, let's leave it in beta for a bit. I'll be working on EAB management in the product too, so I'll revisit this soon.